text_adventure.py

```diff
@@ -0,0 +1,224 @@
+'''
+This is an example text-based adventure! Play it multiple times to get all the endings and fight randomized bosses
+'''
+
+from random import randint, choice
+from time import sleep
+import os
+# from colorama import Fore, Style
+
+# You can change these constant lists depending on the story you create
+yes_no = ["yes", "no"]
+directions = ["left", "right"]
+moves = ["fight", "run", "dodge", "inspect"]
+classes = ["knight", "hunter"]
+
+def make_decision(prompt, choices):
+  """
+  Returns any valid decision the user makes
+  """
+  while True:
+    user_input = input(prompt).lower()
+    if user_input in choices:
+      return user_input
+    else:
+      print("Invalid choice, try again!")
+
+def generate_player():
+  """
+  Returns randomly generated stats depending on the chosen class
+  """
+  player_class = make_decision("> Are you a knight or a hunter?: ", classes)
+  sleep(1)
+  if player_class == 'knight':
+    print("> You are a knight! You are endowed with strength and courage, but you are extremely stubborn!")
+    return {
+      "hp": randint(100, 120),
+      "atk": randint(5, 10),
+      "spd": randint(10, 55)
+    }
+  else:
+    print("> You are a Hunter! You are endowed with agility and stealth, but you lack vitality!")
+    return {
+      "hp": randint(70, 100),
+      "atk": randint(7, 10),
+      "spd": randint(35, 60)
+    }
+  
+def generate_boss():
+  """
+  Returns a randomly generated boss with stats
+  """
+  boss_type = choice(["dragon", "orc"])
```

Comment:
This list of bosses must be moved to the top with the rest of the constant lists. This is extremely important and necessary to bring up, as people with OCD viewing this may collapse from distress upon seeing this.

Reply:
I extremely agree